package.json

```diff
@@ -13,7 +13,7 @@
     "build:preview": "yarn build:env-loader && node ./scripts/yarn/build:preview.js",
     "build:sitemap": "yarn build:env-loader & node ./scripts/yarn/build:sitemap.js",
     "start": "yarn build:env-loader && node ./scripts/yarn/start.js",
-    "export": "yarn build:env-loader && node ./scripts/yarn/export.js",
+    "export": "yarn build:env-loader && node ./scripts/yarn/export.js --USE_REDIS=true",
```

Comment:
This makes sense. I think I like it.

Notably, it's inconsistent with the way we're doing some other things like the following example:
```
"build": "yarn build:env-loader && node ./scripts/yarn/build.js",
```
```
const { processEnv } = require('env-loader')

processEnv('SSG=true next build')
```

I think I like this better, though, and am likely going to push up a PR to bring the above example in line with this approach because I think it's useful to be able to set these commands via CLI. I have, for example, needed to comment out the check for SSG on `yarn dev` because I wanted to test some things. I think running `yarn dev --SSG=false` or `yarn dev --no-SSG` would be nice.  See my other comment on the syntax question I just alluded to.